packages/cactus-plugin-keychain-memory/src/main/typescript/prometheus-exporter/prometheus-exporter.ts

```diff
@@ -0,0 +1,40 @@
+import promClient from "prom-client";
+import { KeyCount } from "./response.type";
+import { collectMetrics } from "./data-fetcher";
+
+export const K_CACTUS_KEYCHAIN_MEMORY_TOTAL_KEY_COUNT =
```

Comment:
I have added this string to the metrics file itself because it seems to be placed there logically (as the string is used first there and the file name is similar to what it contains, i.e. metrics.ts)
Please let me know if it will be good to have the string there or should I create a file just for the string